app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -73,7 +73,7 @@ const isNumber = (val: number) => val || val === 0
 // 100, 100, false, "%" --> 100%
 export const getRangeString = (min: number, max: number, currency?: boolean, suffix?: string) => {
   // If min is 0, return "up to {max}"
```

Comment:
since the comparison logic has changed, we should update this comment to mention the `-1` scenario

Reply:
Good catch, thank you